tg.js

```diff
@@ -57,12 +57,12 @@ generic.username Username
 generic.password  Illagay ang password
 generic.register Mag-parehistro
 generic.login Mag log-in

```

Comment:
Could it be 'Mag Log-in'?

Reply:
I committed it, with the other suggestions of yours, Thanks, did some changes to make it shorter.